lib/string_reverse.rb

```diff
@@ -1,4 +1,25 @@
 # A method to reverse a string in place.
 def string_reverse(my_string)
-  my_string << "not implemented"
+
+  if my_string == nil
+    return nil
+  end
+
+  length = my_string.length
+
+  i = 0
+  j = length - 1
+
+  first = my_string[i]

```

Comment:
Think through lines 13 through 23. How can you make this more consolidated and use only one temporary variable for swapping? Give it some thought, then compare with https://github.com/Ada-C10/string_reverse/blob/solution/lib/string_reverse.rb Slack me if you'd like to discuss it.